scipy/special/tests/test_extending.py

```diff
@@ -7,6 +7,7 @@
 from scipy.special import beta, gamma
 
 
+@pytest.mark.xslow
```

Comment:
This one is not okay. This is basic test coverage for the `cython_special` module.

Reply:
I made them all xslow knowing that everyone would be pinged as a code owner, and I would be corrected 
It worked! We'll take it out of this PR and give it more time to run in the fail_slow PR.

Reply:
```suggestion
```